legal-api/src/legal_api/version.py

```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '2.154.1'  # pylint: disable=invalid-name
+__version__ = '2.154.2'  # pylint: disable=invalid-name
```

Comment:
I know... I'm updating the version in this branch named after the previous version, but I don't think we need another release branch, do we?